windows/internal/install_nightly_package.bat

```diff
@@ -0,0 +1,70 @@
+if "%PACKAGE_TYPE%" == "wheel" goto wheel
+if "%PACKAGE_TYPE%" == "conda" goto conda
+if "%PACKAGE_TYPE%" == "libtorch" goto libtorch
+
+:wheel
+echo "install pytorch wheel from nightly"
+
+pip install --upgrade pip -q
+if errorlevel 1 exit /b 1
+
+set pip_url="https://download.pytorch.org/whl/nightly/%DESIRED_CUDA%/torch_nightly.html"
+if "%DESIRED_CUDA%" == "cu102" (
+    set package_name_and_version="torch==%NIGHTLIES_DATE_PREAMBLE%%DATE%"
+) else (
+    set package_name_and_version="torch==%NIGHTLIES_DATE_PREAMBLE%%DATE%+%DESIRED_CUDA%"
+)
+pip install "%package_name_and_version%" -f "%pip_url%" --no-cache-dir --no-index -q
+if errorlevel 1 exit /b 1
+
+exit /b 0
+
+:conda
+echo "install pytorch conda from nightly"
+set package_name_and_version="pytorch==%NIGHTLIES_DATE_PREAMBLE%%DATE%"
+
+if "%DESIRED_CUDA%" == "cpu" (
+    call conda install -yq -c pytorch-nightly %package_name_and_version% cpuonly
+) else (
+    call conda install -yq -c pytorch-nightly "cudatoolkit=%CUDA_VERSION_STR%" %package_name_and_version%
+)
+if ERRORLEVEL 1 exit /b 1
+
+FOR /f %%i in ('python -c "import sys;print(sys.version)"') do set cur_python=%%i
+
+if not %cur_python:~0,3% == %DESIRED_PYTHON% (
+    echo "The Python version has changed to %cur_python%"
+    echo "Probably the package for the version we want does not exist"
+    echo "conda will change the Python version even if it was explicitly declared"
+)
+
+if "%DESIRED_CUDA%" == "cpu" (
+    call conda list torch | grep cuda || exit /b 0
```

Comment:
nit: Consider replacing `grep` with `findstr`.

Reply:
Why do you search for cuda here? Could you please explain?

Reply:
> nit: Consider replacing `grep` with `findstr`.

ok

Reply:
> Why do you search for cuda here? Could you please explain?

If 'cuda' is found in torch package name, then it's build for cuda, not for cpu, print the error info and `exit /b 1`, else `exit /b 0`